doc/api/style_api.rst

```diff
@@ -2,20 +2,33 @@
 ``matplotlib.style``
 ********************
 
-.. seealso::
+The ``style`` module adds support for easy-to-switch plotting "styles" with the 
+same parameters as a matplotlib rc file that is read at startup to configure 
+matplotlib.
 
-   Examples of using style sheets with :func:`matplotlib.style.use` can be found at :doc:`/gallery/style_sheets/style_sheets_reference`.
+:doc:`/tutorials/introductory/customizing` describes the mechanism and usage
+of styles.
+
+The :doc:`/gallery/style_sheets/style_sheets_reference` gives an overview of
+the builtin styles.
 
```

Comment:
I added the pre-amble above, and removed the "see also".  I agree that the bulk of the documentation can be in the tutorial, but I think its helpful to say what the module does in a sentence too before sending the user elsewhere.

Reply:
BTW, hope it was OK to push to your branch.  And feel free to revert if you disagree....

Reply:
That's ok (even though I find that wording not particularly great).